drivers/overlay/check.go

```diff
@@ -145,7 +145,7 @@ func doesMetacopy(d, mountOpts string) (bool, error) {
 		opts = fmt.Sprintf("%s,%s", opts, data)
 	}
 	if err := unix.Mount("overlay", filepath.Join(td, "merged"), "overlay", uintptr(flags), opts); err != nil {
-		return false, errors.Wrapf(err, "failed to mount overlay for metacopy check with %q options", mountOpts)
+		logrus.Warnf("failed to mount overlay for metacopy check with %q options : %v", mountOpts, err)
```

Comment:
Does this actually fix it?  IE Is the option removed when this happens?  So that a future mount will not use the metacopy=on option?